alembic/env.py

```diff
@@ -0,0 +1,82 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from database import Base
+from cities.models import City
```

Comment:
Ensure that the `City` model is correctly imported from the `cities.models` module. If `City` is not defined or incorrectly imported, it might cause issues during migrations.